tools/biohansel/biohansel.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="biohansel" name="biohansel" version="2.1.3">
+<tool id="biohansel" name="biohansel" version="2.1.0">
```

Comment:
This number needs to be increased, not decreased :(
Maybe 2.1.3.1 for the time being and sync the versions as soon as hansel is released at 2.2?

Reply:
Phil told me to leave it like that but I can easily increase it